cypress/support/fragments/orders/basicOrderLine.js

```diff
@@ -52,5 +56,74 @@ export default {
       instructions: '',
       vendorAccount: '1234'
     }
+  },
+  specialOrderLine: {
```

Comment:
could you add comment with details of defined test objects? It can help to determine when we should use existing object and when we should create new or update something based on these objects.